tests/api/getBaldiyatsForDairaByCodeForWilayaByCode.test.js

```diff
@@ -0,0 +1,69 @@
+describe('get matching baldiyats  by  daira code and wilaya mattricule', ()=>{
+  const mockData = [
+    {
+      mattricule: 47,
+      name: 'Ghardaïa',
+      dairats: [
+        {
+          code: 4701,
+          name: 'GHARDAIA',
+          name_ar: 'غرداية',
+          name_en: 'GHARDAIA',
+          baladyiats: [
+            {
+              code: 4701,
+              name: 'GHARDAIA',
+              name_en: 'GHARDAIA',
+              name_ar: 'غرداية',
+            },
+            {
+              code: 4702,
+              name: 'EL MENIAA',
+              name_en: 'EL MENIAA',
+              name_ar: 'المنيعة',
+            },
+          ],
+        },
+      ],
+    },
+  ];
+
+  let getBaldiyatsForDairaByCodeForWilayaByCode;
+
+  beforeEach(()=>{
+    getBaldiyatsForDairaByCodeForWilayaByCode = require('../../src/api/getBaldiyatsForDairaByCodeForWilayaByCode');
+  });
+
+
+
+  it('should export a function', () => {
```

Comment:
Please apply the linter+prettier to remove such whitespaces and to add an empty line at EOF